packages/themes/src/themes/console/dark.ts

```diff
@@ -371,7 +371,7 @@ export const darkTheme = {
   theme: 'dark',
   typography: {
     body: {
-      fontFamily: 'Inter',
+      fontFamily: 'Inter, sans-serif',
```

Comment:
As written on top of each files those are generated based on what design send us back from figma tokens. If you want to change this you need to change it from figma directly or see with a designer to do so.

Reply:
I didn't check properly the top of the file. I will have a look with designer or eventually check the script token generation